gnd/src/main/java/com/google/android/gnd/persistence/local/room/RoomLocalDataStore.java

```diff
@@ -585,4 +586,32 @@ public Completable insertOrUpdateOfflineArea(OfflineBaseMap area) {
         .toSingle()
         .subscribeOn(schedulers.io());
   }
+
+  @Override
+  public Completable deleteOfflineArea(String id) {
+    return offlineBaseMapDao
+        .findById(id)
+        .toSingle()
+        .doOnSubscribe(__ -> Timber.d("Deleting offline area: %s", id))
+        .flatMapCompletable(offlineBaseMapDao::delete)
```

Comment:
Why do we need to find the area in order to delete it? We should be able to just deleted by id, no?